x11-wm/emwm/emwm-1.0a.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit optfeature
+
+DESCRIPTION="Enhanced Motif Window Manager (MWM fork). Supports Xinerama, Xrandr, UTF-8/Xft, and relevant Extended Window Manager Hints (EWMH)."
+HOMEPAGE="http://fastestcode.org/emwm.html"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/alx210/emwm.git"
```

Comment:
Nit: I'd swap these two, even though EGIT_REPO_URI is not a pre-inherit variable, it feels cleaner.

Example: imagine if, in future, it wanted to warn on git:// URLs and die immediately. Having it the other way around would facilitate that.